### ISSUE TITLE
Warning when using `AnimationParameter` on a disabled GameObject.

### ISSUE THREAD

**Issue by Lawendt** [bitcake/bitstrap#5]
This warning is called whenever there is a `AnimationParameter` on a disabled GameObject:

```
Animator is not playing an AnimatorController
UnityEngine.Animator:get_parameterCount()
BitStrap.AnimationParameterDrawer:OnGUI(Rect, SerializedProperty, GUIContent) (at Assets/Scripts/BitStrap/Plugins/Editor/Animation/AnimationParameterDrawer.cs:36)
UnityEditor.Editor:DrawDefaultInspector()
```

**Comment by vamolessa** [comment]
Fixed on 175029a51d2f959050f364ee33f81972517a255c

Although the solution is *not* displaying the dropdown when the GO is inactive or when there's no AnimatorController.

I'm not yet sure if there's a way to query Animator params when the GameObject is not active.

**Comment by vamolessa** [comment]
I'll close this for now
